scripts/jenkins/jobs-obs/openstack-trackupstream.yaml

```diff
@@ -155,6 +157,8 @@
               "openstack-horizon-plugin-ironic-ui",
               "openstack-horizon-plugin-magnum-ui",
               "openstack-octavia",
+              "openstack-murano",
+              "python-muranoclient",
```

Comment:
There is no python-muranoclient in Cloud:OpenStack:Mitaka yet, but this would try to update it. Either move this line to the 1st section or create the Mitaka package.